src/app/pages/app-list/app-list.component.scss

```
@@ -1111,6 +1115,49 @@
   }
 
+
+  .badges-container {
+    position: absolute;
+    top: 10px;
+    inset-inline-start: 10px; // ✅ LTR: left | RTL: right
+    display: flex;
+    flex-direction: column;
+    gap: 6px;
+    z-index: 5;
+    // Don’t block clicking the card/routerLink
```

Stylelint (17.3.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)